keepassxc-browser/background/client.js

```diff
@@ -176,6 +184,11 @@ keepassClient.getNonce = function() {
     return nacl.util.encodeBase64(nacl.randomBytes(keepassClient.keySize));
 };
 
+// Creates a random number for Request ID
+keepassClient.getRequestId = function() {
+    return String(Math.floor(Math.random() * Number.MAX_SAFE_INTEGER));
```

Comment:
This could generate a large string. Sorry to go back and forth, maybe this method is better: `Math.random().toString(16).substr(2, 8); // 6de5ccda`

Reply:
I agree. Let's do that.

Reply:
Fixed. Though the correct format for the generation is `Math.random().toString(16).substring(2, 10);` because `substr()` is deprecated.